packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -438,7 +438,7 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
 
     addAutoDisposeListener(selectionNotifier, () {
       setState(() {
-        final node = selectionNotifier.value.node;
+        final node = selectionNotifier!.value.node;
```

Comment:
you should be able to remove all these null assertions `!` if `selectionNotifier` is non-nullable